course/templates/course/create_review.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'base_home.html' %}
+{% load static %}
+{% block content %}
+    <title>Course Review</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
```

Comment:
It seems like the base_home links bootstarp v5.0.0 so maybe no need for this link (which is v4.0.0)?

Reply:
Yes, I removed it